lib/common.sh

```diff
@@ -206,7 +210,11 @@ equals(){
 # - String to look for the substring in
 #
 is_in(){
-  [[ "${2}" == *"${1}"* ]]; process_status "$?"
+  [[ "${2}" == *"${1}"* ]]; RET_CODE="$?"
+  if ! process_status "$?" ; then
```

Comment:
This needs to be process_status $RET_CODE, $? here will always be 0 because the RET_CODE assignment is never going to fail:

```
$ cat noexist
cat: noexist: No such file or directory
$ echo $?
1

$ cat noexist; FOO=$?
cat: noexist: No such file or directory
$ echo $?
0
```